service/resource/cloudformation/template.go

```diff
@@ -1,18 +1,22 @@
 package cloudformation
 
 const (
-	MainTemplate = `AWSTemplateFormatVersion: 2010-09-09
-Description: {{ .ASGType }} autoscaling group
+	MainTemplate = `{{define "main"}}AWSTemplateFormatVersion: 2010-09-09
+Description: Main CloudFormation stack.
 Resources:
-  {{ .ASGType }}LaunchConfiguration:
+  {{template "launch_configuration" .}}
+  {{template "autoscaling_group" .}}
+{{end}}`
+
+	LaunchConfigurationTemplate = `{{define "launch_configuration"}}{{ .ASGType }}LaunchConfiguration:
```

Comment:
Using go files for the templates is good for now. But I think it would be good to switch to separate template files as it makes them easier to edit. WDYT?

The downside with using template files is we need to add them to the docker image.

Reply:
Very nice, I was thinking of having separate go files for each resource but it was kind of weird, I'll prepare the template files and the changes to the Dockerfile to add them to the image.

Reply:
Great!  I think this is much better than having huge template files like in k8scloudconfig.

Reply:
Done, in fact the Dockerfile already had it in place :) I've added a relative reference in `main_stack.go`, it works fine in the unit tests, will do a quick check on the container but should work the same.